vtex/checkout-ui-tests
# CHK-811: Add geolocation test for paraguay without postal code

#### What is the purpose of this pull request?

Adds a test for Paraguay which uses our geolocation account and an address that 
doesn't have a postal code (the shipping SLAs are calculated based on the geo 
coordinates of the address).

#### What problem is this solving?

Makes sure we don't break our stores which relies on the geo coordinates of the 
address, and don't care about the address postal code.

#### How should this be manually tested?

Run `yarn cypress open` and run the "Delivery - Paraguay - vtexgame1geo" test, 
it should pass.

#### Screenshots or example usage

#### Types of changes
